src/codewhisperer/service/recommendationHandler.ts

```diff
@@ -240,8 +240,7 @@ export class RecommendationHandler {
             if (latency === 0) {
                 latency = startTime !== 0 ? performance.now() - startTime : 0
             }
-            getLogger().error('CodeWhisperer Invocation Exception : ', error)
-            getLogger().verbose(`CodeWhisperer Invocation Exception : ${error}`)
+            getLogger().error('CodeWhisperer Invocation Exception : %s', (error as Error).message)
```

Comment:
stringifying the entire error usually isn't useful, the stacktrace is mangled in production. Therefore it's common to only mention the  `message` field .